src/Extension/CoreExtension.php

```diff
@@ -915,6 +916,20 @@ public static function keys($array): array
         return array_keys($array);
     }
 
+    /**
+     * Invokes a callable
+     *
+     * @param callable $callable
+     * @param ...$arguments
+     * @return mixed
```

Comment:
phpdoc is not added when it does not provide information beyond the native types.

```suggestion
```